### PR TITLE
Rings stuff and whetstone

### PR DESCRIPTION
**Pull request by Luctus-Comptus**
rings suck if they arnt dragon or null. made some ones for peasants and churchy people

used sinner sprite, made lighter ring. mini torch/lantern (smithable)

made a shitty coder sprite psi ring for church. lets them do there miracles as a focus (smithable)

there was a whetstone sprite I seen in my travels. I used it. 2 stones to make. the math is 25% sharp restore on swipe... roughly

here is sprite they made it for me originally but I share. because hoarding cool things is only acceptable when its a reskin or soemthing

image

the lead pipe sprites mine tho. I need that for things.

### CODE REVIEW

File: code/game/objects/items/rogueitems/natural/stones.dm [hunk 1]
```diff
@@ -330,6 +330,39 @@ GLOBAL_LIST_INIT(stone_personality_descs, list(
 		return
 	..()
 
+
+/obj/item/natural/whet
+	name = "whet stone"
```
Comment (Nakalon) [inline code/game/objects/items/rogueitems/natural/stones.dm:335]: whet - wet typo

File: code/game/objects/items/rogueitems/natural/stones.dm [hunk 2]
```diff
@@ -330,6 +330,39 @@ GLOBAL_LIST_INIT(stone_personality_descs, list(
...
+	icon_state = "whet"
+	desc = "A piece of rough smoothed down stone, better blade maintinance"
```
Comment (Nakalon) [inline code/game/objects/items/rogueitems/natural/stones.dm:337]: maintinance - maintenance typo

File: code/modules/clothing/rogueclothes/rings.dm [hunk 3]
```diff
@@ -18,11 +18,54 @@
 	icon_state = "ring_s"
 	sellprice = 33
 
+/obj/item/clothing/neck/roguetown/psicross/ring   /////////////////////// cast focus ring for acolytes and shit.
```
Comment (Nakalon) [inline code/modules/clothing/rogueclothes/rings.dm:21]: Why is this a neck item if its a ring?

File: code/modules/clothing/rogueclothes/rings.dm [hunk 4]
```diff
@@ -18,11 +18,54 @@
...
+	name = "psi ring"
+	desc = "a blessed psi ring, for the religious type"
+	w_class = WEIGHT_CLASS_TINY
+	icon = 'icons/roguetown/clothing/rings.dmi'
+	mob_overlay_icon = 'icons/roguetown/clothing/onmob/rings.dmi'
+	sleeved = 'icons/roguetown/clothing/onmob/rings.dmi'
+	sleevetype = "shirt"
```
Comment (Nakalon) [inline code/modules/clothing/rogueclothes/rings.dm:28]: Why does it have sleevetype 😭

File: code/modules/roguetown/roguecrafting/items.dm [hunk 5]
```diff
@@ -10,6 +10,12 @@
 				/obj/item/natural/fibers = 1)
 	craftdiff = 0
 
+/datum/crafting_recipe/roguetown/whet
+	name = "whet stone"
```
Comment (Nakalon) [inline code/modules/roguetown/roguecrafting/items.dm:14]: whet to wet

File: code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/valuables.dm [hunk 6]
```diff
@@ -1,3 +1,25 @@
+
+/datum/anvil_recipe/valuables/special/ringf/obj/item/clothing/neck/roguetown/psicross
```
Comment (Nakalon) [inline code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/valuables.dm:2]: something freaky going on here

File: roguetown.dme [hunk 7]
```diff
@@ -2315,8 +2315,6 @@
 #include "code\modules\jobs\job_types\roguetown\mercenaries\types\desertrider.dm"
 #include "code\modules\jobs\job_types\roguetown\mercenaries\types\forlorn.dm"
 #include "code\modules\jobs\job_types\roguetown\mercenaries\types\grenzelhoft.dm"
-#include "code\modules\jobs\job_types\roguetown\mercenaries\types\grenzelhoft_kommandant.dm"
-#include "code\modules\jobs\job_types\roguetown\mercenaries\types\guild_clerk.dm"
```
Comment (Nakalon) [inline roguetown.dme:2319]: why are there being excluded?

**Review by Nakalon — CHANGES_REQUESTED** [review]

**Comment by Nakalon** [comment]
Also, test evidence please!

File: code/modules/clothing/rogueclothes/rings.dm (hunk 3 above)
Comment (Nakalon) [reply code/modules/clothing/rogueclothes/rings.dm:21]: Why not... uh.. that seems like it should be fixed if thats the case on other rings and not piled onto, makes it extremely confusing

File: code/game/objects/items/rogueitems/natural/stones.dm (hunk 1 above)
Comment (Nakalon) [reply code/game/objects/items/rogueitems/natural/stones.dm:335]: should be whetstone then

**Comment by JohnnyJohnJohnson** [comment]
These sound awesome, they should be included.

**Pull request closed without merging**